package.json

```diff
@@ -132,6 +134,7 @@
     "nodemon": "^2.0.4",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
+    "otpauth": "^6.0.4",
```

Comment:
What is the difference between `otpauth` and `speakeasy`?

Reply:
`speakeasy` creates the long secret that you scan in the authenticator app and store in the db. Then the authenticator app creates TOTP codes (the 6-digits) based on the secret.

`otpauth` also creates TOTP codes from a secret.

So they are two sides of the same coin.